.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+name: ci
+
+env:
+  RUST_BACKTRACE: 1
+
+# By default depandabot only receives read permissions. Explicitly give it write
+# permissions which is needed by the ouzi-dev/commit-status-updater task.
+#
+# Updating status is relatively safe (doesnt modify source code) and caution
+# should we taken before adding more permissions.
+permissions:
+  statuses: write
```

Comment:
How does dependabot come into play here? Isn't this about the regular CI workflow? I fail to see the connection, to be honest.